apps/route_patterns/lib/repo.ex

```diff
@@ -43,6 +43,12 @@ defmodule RoutePatterns.Repo do
     |> Enum.sort(&reorder_mrts(&1, &2, route_id))
   end
 
+  def by_stop_id(stop_id, opts) do
```

Comment:
suggestion: I don't think `opts` is used, so you could remove it:
```suggestion
  def by_stop_id(stop_id) do
```